examples/lb/lb.go

```diff
@@ -11,9 +11,11 @@ type LB struct {
 	id int
 }
 
-// Reduce implements the `phi.Reducer` interface. We simulate a slow task by
+// Handle implements the `phi.Handlerr` interface. We simulate a slow task by
```

Comment:
Typo: should be `phi.Handler`
```suggestion
// Handle implements the `phi.Handler` interface. We simulate a slow task by
```